wgpu/src/lib.rs

```diff
@@ -3,55 +3,7 @@
 //! To start using the API, create an [`Instance`].
 //!
 //! ## Feature flags
-// NOTE: feature docs. below should be kept in sync. with `Cargo.toml`!
-//!
-//! ### Backends
-//!
-//! ⚠️ WIP: Not all backends can be manually configured today. On Windows & Linux the **Vulkan & GLES
-//! backends are always enabled**. See [#3514](https://github.com/gfx-rs/wgpu/issues/3514) for more
-//! details.
-//!
-//! - **`dx12`** _(enabled by default)_ ---   Enables the DX12 backend on Windows.
-//! - **`metal`** _(enabled by default)_ --- Enables the Metal backend on macOS & iOS.
-//! - **`webgpu`** _(enabled by default)_ --- Enables the WebGPU backend on Wasm. Disabled when targeting `emscripten`.
-//! - **`angle`** --- Enables the GLES backend via [ANGLE](https://github.com/google/angle) on macOS.
-//! - **`vulkan-portability`** --- Enables the Vulkan backend on macOS & iOS.
-//! - **`webgl`** --- Enables the GLES backend on Wasm.
-//!     - ⚠️ WIP: Currently will also enable GLES dependencies on any other targets.
-//!
-//! **Note:** In the documentation, if you see that an item depends on a backend,
-//! it means that the item is only available when that backend is enabled _and_ the backend
-//! is supported on the current platform.
```

Comment:
todo: I don't see this paragraph in any TOML documents, ATM. Need to track this down in history.

Reply:
This was me - I added it directly to the Readme. I didn't realize we were planning on going back.

Reply:
I mean, we could! If everybody is fine with that, I can drop this PR. 😊 My intent here was to give the community back the nice thing that Firefox couldn't have for a while.

Reply:
oh please let's go back. Imho it's way better with `document-features`

Reply:
@cwfitzgerald: uh

I think you mean crate documentation instead? 🤔

Reply:
I've reconciled differences between the current contents of `wgpu/Cargo.toml` and `lib.rs` with fb6289158 and 328bf5551. Differences of note:

* The `**Note:**` paragraph this conversation is attached to was missing from `Cargo.toml`.
* The `serde` feature's documentation was missing from `lib.rs`.
* The `webgpu` feature was in different positions between them.

…thus demonstrating that a single source of truth here is already going to be beneficial, despite the short window of time in which these two documents diverged. 😅

I also validated that `wgpu-core/Cargo.toml` and `wgpu-core/src/lib.rs` did not diverge meaningfully.